### PR TITLE
Add board

### PR DESCRIPTION
**Pull request by gabafior**

### CODE REVIEW

File: client/src/main/java/com/pk/App.java [hunk 1]
```diff
@@ -1,36 +1,70 @@
-package com.pk;
+package com.pk.checkers;
```
Comment (ronbun) [inline client/src/main/java/com/pk/App.java:1]: This change is not needed (and is wrong)

File: client/src/main/java/com/pk/checkers/Piece.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package com.pk.checkers;
+
+import javafx.scene.layout.StackPane;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Ellipse;
+import javafx.scene.shape.SVGPath;
+
+import static com.pk.checkers.CheckersApp.TILE_SIZE;
```
Comment (ronbun) [inline client/src/main/java/com/pk/checkers/Piece.java:8]: Change to import static com.pk.App.TILE_SIZE;

File: client/src/main/java/com/pk/checkers/Piece.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+public class Piece extends StackPane {
+
+    private PieceType type;
+
+    private double mouseX, mouseY;
+    private double oldX, oldY;
```
Comment (ronbun) [inline client/src/main/java/com/pk/checkers/Piece.java:16]: Multiple variables should not be declared on the same line

File: client/src/main/java/com/pk/checkers/Piece.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        
+
+        setOnMousePressed(e -> {
+            mouseX = e.getSceneX();
+            mouseY = e.getSceneY();
+        });
+
+        setOnMouseDragged(e -> {
+            relocate(e.getSceneX() - mouseX + oldX, e.getSceneY() - mouseY + oldY);
+        });
```
Comment (ronbun) [inline client/src/main/java/com/pk/checkers/Piece.java:67]: {} are not needed here

File: client/src/main/java/com/pk/checkers/Tile.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+    public void setPiece(Piece piece) {
+        this.piece = piece;
+    }
+
+    public Tile(boolean light, int x, int y) {
+        setWidth(CheckersApp.TILE_SIZE);
+        setHeight(CheckersApp.TILE_SIZE);
+
+        relocate(x * CheckersApp.TILE_SIZE, y * CheckersApp.TILE_SIZE);
```
Comment (ronbun) [inline client/src/main/java/com/pk/checkers/Tile.java:27]: CheckersApp -> App

File: client/src/main/java/com/pk/App.java [hunk 6]
```diff
@@ -1,36 +1,70 @@
...
-  }
-
-  public static void main(String[] args) {
-    launch(args);
-  }
+    public static final int TILE_SIZE = 100;
+    public static final int WIDTH = 8;
+    public static final int HEIGHT = 8;
+
+    private Tile[][] board = new Tile[WIDTH][HEIGHT];
```
Comment (ronbun) [inline client/src/main/java/com/pk/App.java:16]: Add Tile import

File: client/src/main/java/com/pk/App.java [hunk 7]
```diff
@@ -1,36 +1,70 @@
...
+            for (int x = 0; x < WIDTH; x++) {
+                Tile tile = new Tile((x + y) % 2 == 0, x, y);
+                board[x][y] = tile;
+
+                tileGroup.getChildren().add(tile);
+
+                Piece piece = null;
+
+                if (y <= 2 && (x + y) % 2 != 0) {
+                    piece = makePiece(PieceType.BLACK, x, y);
```
Comment (ronbun) [inline client/src/main/java/com/pk/App.java:36]: Add PieceType import

File: client/src/main/java/com/pk/App.java [hunk 8]
```diff
@@ -1,36 +1,70 @@
...
+    public void start(Stage primaryStage) throws Exception {
+        Scene scene = new Scene(createContent());
+        primaryStage.setTitle("CheckersApp");
+        primaryStage.setScene(scene);
+        primaryStage.show();
+    }
+
+    private Piece makePiece(PieceType type, int x, int y) {
+        Piece piece = new Piece(type, x, y);
+        return piece;
```
Comment (ronbun) [inline client/src/main/java/com/pk/App.java:64]: Return it directly

File: client/src/main/java/module-info.java [hunk 9]
```diff
@@ -1,20 +1,23 @@
...
-  exports com.pk.lanserver;
-  exports com.pk.lanserver.exceptions;
-  exports com.pk.lanserver.models;
-  exports com.pk.frontend.menu;
-}
+    requires javafx.controls;
+    requires javafx.graphics;
+    requires javafx.fxml;
+    requires javafx.swing;
+    requires javafx.media;
```
Comment (ronbun) [inline client/src/main/java/module-info.java:6]: Are you sure that media and swing are required? I am able to run it without them.

File: client/src/main/java/module-info.java [hunk 10]
```diff
@@ -1,20 +1,23 @@
...
-
-  exports com.pk;
-  exports com.pk.logic;
-  exports com.pk.logic.exceptions;
-  exports com.pk.lanserver;
-  exports com.pk.lanserver.exceptions;
-  exports com.pk.lanserver.models;
-  exports com.pk.frontend.menu;
-}
+    requires javafx.controls;
```
Comment (ronbun) [inline client/src/main/java/module-info.java:2]: Duplicated controls, delete one without "transitive" keyword

**Review by ronbun — CHANGES_REQUESTED** [review]
+ Change in your IDE settings indentation from 4 spaces to 2 spaces.

File: client/src/main/java/com/pk/App.java (hunk 6 above)
Comment (Rogrir) [reply client/src/main/java/com/pk/App.java:16]: To add to that best would be to change it to List

**Review by ronbun — APPROVED** [review]
Looks good

**Review by Rogrir — APPROVED** [review]

**Pull request merged**